shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DALStatement.g4

```diff
@@ -33,31 +33,7 @@ explain
     | explainType? (explainableStatement | FOR CONNECTION connectionId)
     | ANALYZE (FORMAT EQ_ TREE)? select)
     ;
-
-showDatabases
```

Comment:
@totalo Why remove these parse rule?

Reply:
It is not deleted, but rearranged in the order of the official documents to facilitate subsequent proofreading.

Reply:
Ok, I got it.